xtable-api/src/main/java/org/apache/xtable/model/metadata/TableSyncMetadata.java

```diff
@@ -51,6 +51,8 @@ public class TableSyncMetadata {
   /** Property name for the XTABLE metadata in the table metadata/properties */
   public static final String XTABLE_METADATA = "XTABLE_METADATA";
 
+  public static final String XTABLE_SOURCE_METADATA = "XTABLE_SOURCE_METADATA";
```

Comment:
Instead of having a separate metadata, is it possible to embed this within the metadata object that we have today?

Reply:
Good catch! The current metadata is table-level information, while the new one is commit-level, but it is true that we can reuse this key.

Reply:
And maybe we could consider moving the source format to table-level information and leaving the source identifier at the commit level to reduce redundancy.

Reply:
Ah I see your point, that's a good way. I’ll get started on it.